service/build.gradle

```diff
@@ -297,7 +297,7 @@ integration {
 }
 
 pitest {
-  pitestVersion.set('1.4.11')
+  pitestVersion.set('1.6.2')
```

Comment:
Can we make sure we test this on the nightly-dev first. I think I remember having to pin it to 1.4.11 as it kept crashing in the pipeline

Reply:
it was tested there
https://build.platform.hmcts.net/blue/organizations/jenkins/HMCTS_Nightly_FPL%2Ffpl-ccd-configuration/detail/nightly-dev/310/pipeline

Reply:
https://build.platform.hmcts.net/job/HMCTS_Nightly_FPL/job/fpl-ccd-configuration/job/nightly-dev/310/changes
commit no 9